src/app.rs

```diff
@@ -519,6 +402,7 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
 
```

Comment:
Can't comment directly on the function (line 395), but possibly move it closer to `bootstrap_vdom`? Since they're closely related.